themes/app/templates/Layout/Page.ss

```diff
@@ -1,4 +1,13 @@
-<h2>$Title</h2>
-$Content
-$Form
-$ElementalArea
+$Breadcrumbs
+<div class="container">
+    <div class="row">
+        <div class="typography col-lg-8">
+            <p class="lead page-content__summary">$Summary</p>
```

Comment:
You can remove `page-content__summary` if it is not being used

Reply:
`.lead` class mighht be enough to style all `summary` blocks

Reply:
Will do 👌🏻